sylt-parser/src/parser.rs

```diff
@@ -1166,8 +1170,8 @@ impl Display for Type {
             TypeKind::Tuple(tys) => {
                 write!(f, "(")?;
                 for (i, ty) in tys.iter().enumerate() {
-                    if i != 0 { write!(f, ", ")?; }
-                    write!(f, "{}", ty)?;
+                    if i != 0 { write!(f, " ")?; }
+                    write!(f, "{},", ty)?;
```

Comment:
I think this causes trailing commas.

Reply:
Yes, but it's acceptable in tuples, I made a formatting decision that you want trailing commas in tuples.

Reply:
It simplifies the implementation for the debug prints. I don't care enough about it to add a special case. If you feel strongly make an issue.